### ISSUE TITLE
Support configuring `http.deregistration_delay` to reduce connection drain period and speed up deployments

### ISSUE THREAD

**Issue by guyschlider** [aws/copilot-cli#2190]
Hey,

When releasing a new version of my service using `svc deploy` I see that the whole process takes around 5 minutes, I suspect that due to the default deregistration_delay value, is there a friendly way to set it in the manifest so it will be set to a lower value?

**Comment by efekarakus** [comment]
Hi @guyschlider !

Unfortunately, we don't have a way at the moment to override the `deregistration_delay` value, Copilot sets the value to `60s` instead of the default `300s` to make deployments faster:
https://github.com/aws/copilot-cli/blob/23bbb9579213e4820db19049fe9b20fb054c07da/templates/workloads/services/lb-web/cf.yml#L167-L168

But overriding this field under `http` would be a new feature request for us.

In the mean time, to speed up deployments it's possible to override the `healthcheck` field (https://aws.github.io/copilot-cli/docs/manifest/lb-web-service/#http-healthcheck), I believe the default values are `30s` for the `interval` and `5` for `healthy_threshold`.  
Updating them to smaller values such as:
```yaml
http:
  healthcheck:
    path: '/'
    healthy_threshold: 2
    interval: 5s
    timeout: 2s
```
should speed things up

**Comment by guyschlider** [comment]
Perfect
🙏 thank you so much

**Comment by Lou1415926** [comment]
The feature is now released in [v1.9.0](https://github.com/aws/copilot-cli/releases/tag/v1.9.0)! 🎊
